zokrates_book/src/gettingstarted.md

```diff
@@ -31,6 +31,40 @@ cargo +nightly build --release
 cd target/release
 ```
 
+If you want to enable the libsnark backend in ZoKrates, you need to install some prerequisites:
```

Comment:
This sounds optional, but since this PR enables libsnark compilation by default, we should probably mark these dependencies as mandatory? Alternatively, we could explicitly document how to build without libsnark, here.

Reply:
Furthermore, specifying minimum versions we tested with could be helpful. Not mandatory.

Reply:
If we're confident enough that MacOS works, it would even make sense to remove the part about building without libsnark entirely? Or maybe it's still useful for other OS?

Reply:
good point, that sounds optimal. only need to document the optional "without-libsnark" build, then.